src/accounts.rs

```diff
@@ -299,9 +302,11 @@ pub const CONFIG_NAME: &str = "accounts.toml";
 /// Database file name.
 pub const DB_NAME: &str = "dc.db";
 
+static CONFIG_FILES: Lazy<Mutex<HashSet<PathBuf>>> = Lazy::new(|| Mutex::new(HashSet::new()));
```

Comment:
Of course we must protect from several processes running on the same configuration, but i think better to do that by locking some lockfile, not `accounts.toml` which can be renamed/removed. I guess there are other reasons to protect from conflicting processes anyway.

As for multiple `Accounts` objects, at least there was a test doing so (which i'm fixing). And also it's possible if an `Accounts` object is recreated for some reason. I have no idea why it can be done, but then it's possible to have two objects for some short period. So, i suggest to limit the scope of this PR to the initial problem (non-atomic update of `accounts.toml`). But maybe a better way to protect from multiple `Accounts` objects is possible. As for protecting from conflicting processes, i'd suggest to solve that separately.

Reply:
If an `Accounts` object will manage the lockfile, looks like `CONFIG_FILES` may be safely removed, yes